### ISSUE TITLE
Add code folding shortcut keys to "fold all" or "fold level N".

### ISSUE THREAD

**Issue by etep** [zed-industries/community#1930]
### Check for existing issues

- [X] Completed

### Describe the feature

Please add shortcut commands and keys (e.g. in the view menu) for cold folding. It would be neat to have "fold all" and "fold to level 1" or "fold to level N" etc.

I am aware of the current code folding shortcut keys, but those only fold the current section of code. This feature would fold every section of code (once or at the Nth level, depending).

Thank you!

**Comment by JosephTLyons** [comment]
I agree that we should have these commands - super useful.  No worries on the duplicate, they can be hard to find.
